### PR TITLE
Metric: Certifications via Caseflow

### PR DESCRIPTION
**Pull request by pkarman**
connects #12554 

### Description
OIT metric for Certifications via Caseflow.

### Testing Plan
1. Monkeypatch and run in production console for verifying logic.

 
### Documentation Updates
- [x] Topline File Descriptions Added or Updated as Needed

### CODE REVIEW

File: app/services/metrics/certification_usage.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  def certified_paperless
+    @certified_paperless ||= certifications.select(&:paperless?).count
+  end
+
+  def certified_with_caseflow
+    @certified_with_caseflow ||= certifications.select(&:certified_with_caseflow?).count
+  end
+
+  def regional_office_codes
```
Comment (pkarman) [inline app/services/metrics/certification_usage.rb]: TBD whether this is needed or not

File: app/models/vacols/case.rb [hunk 2]
```diff
@@ -315,6 +315,14 @@ def remand_return_date(vacols_ids)
     # :nocov:
   end
 
+  def paperless?
+    folder.tivbms == "Y"
+  end
+
+  def certified_with_caseflow?
```
Comment (pkarman) [inline app/models/vacols/case.rb]: https://dsva.slack.com/archives/C2ZAMLK88/p1485890347000118

File: app/services/metrics/certification_usage.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Metric ID: 1709076050
```
Comment (alisan16) [inline app/services/metrics/certification_usage.rb]: Can you update the metric ID to the new ID here: https://github.com/department-of-veterans-affairs/caseflow/issues/12748 I think 17### was from the old contract.

File: spec/services/metrics/certification_usage_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        vacols_case = create(:case, :type_original, bf41stat: 3.days.ago)
+        vacols_case.folder.update!(tivbms: "N")
+      end
+    end
+
+    it "reports total and paperless metrics" do
+      expect(subject).to eq(
+        certified_total: 25,
+        certified_paperless: 15,
+        certified_with_caseflow: 15,
```
Comment (alisan16) [inline spec/services/metrics/certification_usage_spec.rb]: How is this metric 15? Can paper cases be certified with caseflow?

Comment (pkarman) [reply spec/services/metrics/certification_usage_spec.rb]: I added some comments.

Note that for the sake of the code/tests, we are just measuring what's in VACOLS. In a practical real-world sense, no, paper cases would not be certified with caseflow, but afaik nothing prevents that from happening, technically, and VACOLS could reflect that situation if it happened.

**Pull request merged**